python/py-crossbar/Portfile

```diff
@@ -0,0 +1,188 @@
+
+name                py-crossbar
+version             20.8.1
+revision            0
+categories          python net
+platforms           darwin
+license             AGPL-3
+
+homepage            https://crossbar.io
+
+description         Multi protocol WAMP/WebSocket REST/HTTP MQTT application \
+                    router for microservices.
+long_description    {*}${description}
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+checksums           rmd160  c52ae7e750d1ea397c6133d6973d39360d7d7511 \
+                    sha256  c0e7a25c12fd768e8cc9be3f572d0f7399b43089b0396635f2ab609fa2855572 \
+                    size    382515
+
+python.versions     38
+
+if {${subport} ni "${name} crossbar"} {
+
+    # Patches:
+    #
+    #   patch-requirements-min.txt.diff:
+    #       remove maximum version limit for idna and urllib3
+    #
+    #   patch-setup.py.diff:
+    #       avoid installing LICENSE files into the Python site-packages root
+    #
+    patchfiles      patch-requirements-min.txt.diff \
+                    patch-setup.py.diff
+
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:py${python.version}-attrs \
+                    port:py${python.version}-autobahn \
+                    port:py${python.version}-bcrypt \
+                    port:py${python.version}-bitstring \
+                    port:py${python.version}-cbor \
+                    port:py${python.version}-click \
+                    port:py${python.version}-constantly \
+                    port:py${python.version}-cryptography \
+                    port:py${python.version}-h2 \
+                    port:py${python.version}-idna \
+                    port:py${python.version}-incremental \
+                    port:py${python.version}-jinja2 \
+                    port:py${python.version}-lmdb \
+                    port:py${python.version}-mistune \
+                    port:py${python.version}-netaddr \
+                    port:py${python.version}-passlib \
+                    port:py${python.version}-priority \
+                    port:py${python.version}-psutil \
+                    port:py${python.version}-pyubjson \
+                    port:py${python.version}-asn1 \
+                    port:py${python.version}-asn1-modules \
+                    port:py${python.version}-pygments \
+                    port:py${python.version}-pynacl \
+                    port:py${python.version}-openssl \
+                    port:py${python.version}-pyqrcode \
+                    port:py${python.version}-pytrie \
+                    port:py${python.version}-yaml \
+                    port:py${python.version}-sdnotify \
+                    port:py${python.version}-service_identity \
+                    port:py${python.version}-setproctitle \
+                    port:py${python.version}-setuptools \
```

Comment:
You've listed py-setuptools as both a build dependency and a lib dependency but it only needs to be listed once. Usually setuptools is only used at build time, but maybe crossbar also uses it at runtime; I'm not sure.

I recommend sorting the list of dependencies alphabetically.